README.md

```diff
@@ -46,3 +45,7 @@ For example, this error happens in case you set SiteURL starting with `http://`,
 Check system logs. If you find the following message, Matterpoll doesn't support your Mattermost. Please update your Mattermost server.
 
 `RPC call to GetServerVersion API failed: rpc: can't find method Plugin.GetServerVersion	{"plugin_id": "com.github.matterpoll.matterpoll", "source": "plugin_stderr"}`
+
+## Contributing
+
+We welcome contributions of any bug reports, issues, feature requests, feature implementations and pull requests. Feel free to join the [Matterpoll channel](https://pre-release.mattermost.com/core/channels/matterpoll) on the Mattermost community server. 
```

Comment:
I want to add the description about filing new issue.

We welcome contributions of any bug reports, issues, feature requests, feature implementations and pull requests. Feel free **to [file a new issue](https://github.com/matterpoll/matterpoll/issues/new) and** to join the [Matterpoll channel](https://pre-release.mattermost.com/core/channels/matterpoll) on the Mattermost community server.